src/network/netdev/bridge.c

```diff
@@ -137,6 +137,12 @@ static int netdev_bridge_post_create_message(NetDev *netdev, sd_netlink_message
                         return r;
         }
 
+        if (b->fdb_max_learned > 0) {
```

Comment:
So, we do not pass `0`, that contradict with the document.
Maybe, better to introduce a flag, say, `Bridge.fdb_max_learned_set`, and pass `IFLA_BR_FDB_MAX_LEARNED` only when the flag is set.

Reply:
Ah I see. Updated.